src/page/tags.ts

```diff
@@ -80,8 +86,7 @@ export default class Tags {
           });
         // 如果存在 pjax，则刷新 wordCloudElement
         if (sakura.$pjax) {
-          // TODO 虽然添加上了，但似乎没生效
-          sakura.$pjax.refresh(wordCloudElement);
+          sakura.$pjax.refresh(document.querySelector('#tag-wordcloud'));
```

Comment:
这里似乎不需要改动。使用已有的 `wordCloudElement` 变量即可。

```suggestion
          sakura.$pjax.refresh(wordCloudElement);
```